src/dogehouse/__init__.py

```diff
@@ -85,6 +89,15 @@ async def send_message(
             tokens=tokenize_message(message)
         )
 
+    async def delete_message(self, message: Message) -> None:
+        await self._send(CHAT_DELETE, messageId=message.id, userId=message.author.id, deleterId=self.user.id)
```

Comment:
add `assert self.user is not None` above this line, it's needed for static type things for now